echopop/core.py

```diff
@@ -72,6 +72,37 @@
     },
 }
 
+# Name configuration dictionary
+NAME_CONFIG = {
+    "Age": "age",
+    "Cell portion": "fraction_cell_in_polygon",
+    "Frequency": "length_count",
+    "haul": "haul_num",
+    "haul end": "haul_end",
+    "haul start": "haul_start",
+    "Haul": "haul_num",
+    "Latitude": "latitude",
+    "Latitude (upper limit)": "northlimit_latitude",
+    "Latitude of centroid": "centroid_latitude",
+    "Length": "length",
+    "Longitude": "longitude",
+    "Longitude of centroid": "centroid_longitude",
+    "strata": "stratum_num",
+    "Sex": "sex",
+    "Ship": "ship_id",
+    "Spacing": "transect_spacing",
+    "Species_Code": "species_id",
+    "Species_Name": "species_name",
+    "Strata Index": "stratum_num",
```

Comment:
These 2 are indeed different names but represent the same thing, right? Just wanted to make sure that's the case.

Reply:
By "these" two do you mean `Strata Index` and `Species Name` as highlighted here? Or do you mean `Strata Index` and `Stratum` (I'm assuming the latter)? If so, then yep, they are the same thing.

Reply:
Oops, yes I highlighted the wrong lines! I did mean `Strata Index` and `Stratum`. Ok!